pdcupdater/handlers/modules.py

```diff
@@ -17,9 +17,9 @@
 class ModuleStateChangeHandler(pdcupdater.handlers.BaseHandler):
     """ When the state of a module changes. """
 
-    tree_processing_states = set(('done', 'ready'))
+    processing_states = set(('done', 'ready', 'init')) # INIT is only for release variant creation
```

Comment:
The name `tree_processing_states` was intentional -- states where pdc-updater needs to process trees :).